queries.sql

```diff
@@ -8,3 +8,39 @@ SELECT name, escape_attempts from animals WHERE weight_kg>10.5;
 SELECT * from animals WHERE neutered=true;
 SELECT * from animals WHERE name!='Gabumon';
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 and 17.3;
+
+
+BEGIN;
+UPDATE animals SET species = 'unidentified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
```

Comment:
- [ ] @IngDavidTM great work on your queries, you forgot to add a screenshot showing the result of the query after rollback to verify changes were undone.

Reply:
Hi @danmainah, I just change the image with the action required but with the new version of the table. Is there any problem because I no longer have the first version of the table?

Reply:
hi @IngDavidTM it's okay since the main point was to showcase the before and after the changes are made which you have done.